docs(cloudflare): Add waitUntil limitation (#17617)
getsentry/sentry-docs: docs/platforms/javascript/guides/cloudflare/index.mdx

This is expected behavior in the Cloudflare Workers environment and affects all frameworks deployed to Cloudflare Workers, including Next.js, Astro, Remix, and others.

### Missing Spans in `waitUntil()`

> If you're using Cloudflare's [`waitUntil()`](https://developers.cloudflare.com/workers/runtime-apis/handlers/fetch/#contextwaituntil) to run background tasks and spans created inside `waitUntil()` are not showing up in Sentry, this is because the request's root span has already ended by the time the background work runs.

Reviewer: ```suggestion
If you're using Cloudflare's [`waitUntil()`](https://developers.cloudflare.com/workers/runtime-apis/handlers/fetch/#contextwaituntil) to run background tasks, spans created inside `waitUntil()` may not appear in Sentry because the request's root span has already finished before the background work starts.
```
